docs/models.yml

```diff
@@ -223,23 +217,13 @@ user:
     replacement: meeting_id
     fields:
       type: relation
-      to:
-        collection: user
-        field:
-          name: vote_delegations_$_from_ids
-          type: structured-relation
-          replacement: meeting_id
+      to: user/vote_delegations_$_from_ids
```

Comment:
Replacement is missing. The replacement in line 217 refers to line 215. Do you want so see an implicit replacement here?